packages/app/src/app/pages/Sandbox/Editor/Workspace/SSEDownNotice/SSEDownNotice.tsx

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
-import { observer, inject } from 'mobx-react';
 import getTemplate from '@codesandbox/common/lib/templates';
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+
+import { useStore } from 'app/store';
 
 import { Container } from './elements';
 
-const ConnectionNotice = ({ store }) => {
+export const SSEDownNotice = observer(() => {
+  const store = useStore();
```

Comment:
@Saeris I think you should destructur `store`  just like you did in the other files

```js
const {
  editor: { currentSandbox: { template } },
  server: { status },
} = useStore();
```